.github/workflows/validation.yml

```diff
@@ -2,13 +2,25 @@
 name: validation
 
 on:  # yamllint disable-line rule:truthy
-  - push
-  - pull_request
+  pull_request:
+  push:
```

Comment:
Don't we want the validation to run on all pushes, in case people make changes while the pull request is open?

Reply:
The `pull_request` by default also reacts on push events, so if you have PR and push a new code into the branch the workflow is triggered.

The change in workflow triggers should remain the same from our PoV but at the same time, it de-duplicates actions that are triggered by both PR and push events.

https://frontside.com/blog/2020-05-26-github-actions-pull_request/